.prettierignore

```diff
@@ -1 +1,2 @@
 **/*.min.js
+**/*.min.css
```

Comment:
Please configure your editor to always leave a blank line at the end of the files.

```suggestion
**/*.min.css

```